internal/metrics/pidstat.go

```diff
@@ -118,7 +118,7 @@ func stat(pid int, statType string) (*SysInfo, error) {
 			_utime = _history.utime
 		}
 		total := stat.stime - _stime + stat.utime - _utime
-		total = total / clkTck
+		total /= clkTck
```

Comment:
this one is less readable, we prefer to write it in a full format in a sake of consistency, we value consistency a lot: https://github.com/doublecloud/transfer/blob/main/CONTRIBUTING.md#code-style